### PR TITLE
Correct grammar and Improve clarity

### PR DESCRIPTION
**Pull request by asdacosta**
## Because

Correct grammar and Improve clarity

## This PR

* Correct grammar 
* Improve clarity

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/installations/installation_overview.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ### Introduction
 
-The first step for building any website is having the right tools. For us, that means setting up a development environment for writing good code.
+The first step in building any website is having the right tools. For us, that means setting up a development environment for writing good code.
 
 Many online development courses use in-browser code editors or "sandboxes," which give you the tools and programs needed to accomplish the task at hand and nothing else. You'll use some of these sandboxes throughout the early stages of The Odin Project since they're great for getting started quickly. However, the best way to set yourself up for long-term success is to operate in a real development environment.
 
 We won't lie to you: installing packages, editors, and even entire operating systems can be very frustrating. However, having the experience of setting up a development environment to run the code you'll write is an invaluable, real-world skill you'll carry with you for the rest of your career.
 
 ### The installation plan
```
Comment (fortypercenttitanium) [inline foundations/installations/installation_overview.md:9]: Let's make the lesson template linter happy here and make this `### Lesson overview`

Comment (fortypercenttitanium) [reply foundations/installations/installation_overview.md:9]: > > Let's make the lesson template linter happy here and make this `### Lesson overview`
> 
> That heading doesn’t only contain an overview of the tasks to be done, so I think changing that to Lesson Overview will create some inconsistency. However, you can do the changes necessary if you see fit.

We have a linter in place that enforces a certain template. You will see the errors shown at the bottom of this PR.

image


I do not make changes on other peoples' PRs. If you want this merged, please implement the changes I'm requesting.

Comment (asdacosta) [reply foundations/installations/installation_overview.md]: It brought up another lint error.

Comment (fortypercenttitanium) [reply foundations/installations/installation_overview.md]: I'll ignore this one - thanks.

**Pull request merged**